monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -420,8 +420,9 @@ class ReportPageComponent extends AuthComponent {
           <PostBreach data={this.state.report.glance.scanned}/>
         </div>
 
-        <div style={{position: 'relative', height: '80vh'}}>
-          {this.generateReportPthMap()}
+        <div style={{position: 'relative' /*, height: '80vh'*/}}>
+          {/*Disable PTH map until we fix it
+          this.generateReportPthMap()*/}
         </div>
```

Comment:
No need to comment out code. Delete, `git` remembers

Reply:
I'm still not sure what's the best routine for this type of code. Just deleting and forcing future developer to search for this isn't the best solution. When future developer will try to implement pth map, he might not even be aware that this code existed.